sqlargs_test.go

```diff
@@ -1,18 +1,53 @@
-package sqlargs_test
+package sqlargs
```

Comment:
Please keep this `sqlargs_test`.

Reply:
Made it part of `sqlargs` package for testing `stripVendor`.

Reply:
Ah, alright then.